### PR TITLE
Fix doc for transpose, conv3d and batch_norm.

### PR DESCRIPTION
**Pull request by ceci3**
transpose:
https://github.com/PaddlePaddle/Paddle/blob/1.4/python/paddle/fluid/layers/nn.py#L5681 It's - > Its

conv3d:
add parameter num_filters

batch_norm:
introduce how to specific the name of parameters

### CODE REVIEW

**Review by XiaoguangHu01 — DISMISSED** [review]

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -3098,18 +3098,24 @@ def batch_norm(input,
...
         data_layout(string, default NCHW): NCHW|NHWC
         in_place(bool, Default False): Make the input and output of batch norm reuse memory.
         name(string, Default None): A name for this layer(optional). If set None, the layer
             will be named automatically.
-        moving_mean_name(string, Default None): The name of moving_mean which store the global Mean.
+        moving_mean_name(string, Default None): The name of moving_mean which store the global Mean. If it 
+            is set to None, batch_norm will not save mean, otherwise, batch_norm will save global Mean with
+            the string.
         moving_variance_name(string, Default None): The name of the moving_variance which store the global Variance.
+            If it is set to None, batch_norm will not save mean, otherwise, batch_norm will save global Variance
```
Comment (xsrobin) [inline python/paddle/fluid/layers/nn.py]: 这个也可以控制save mean？

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -5998,7 +6004,7 @@ def transpose(x, perm, name=None):
     if len(perm) != len(x.shape):
         raise ValueError(
             "Input(perm) is the permutation of dimensions of Input(input). "
-            "It's length shoud be equal to Input(input)'s rank.")
+            "Its length shoud be equal to Input(input)'s rank.")
```
Comment (xsrobin) [inline python/paddle/fluid/layers/nn.py]: should be

**Review by xsrobin — COMMENTED** [review]
as comments

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (ceci3) [reply python/paddle/fluid/layers/nn.py]: done, thanks

File: python/paddle/fluid/layers/nn.py (hunk 2 above)
Comment (ceci3) [reply python/paddle/fluid/layers/nn.py]: done, thanks

**Review by xsrobin — APPROVED** [review]

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**